test/test_change_stream.py

```diff
@@ -1086,14 +1050,66 @@ def setUp(self):
         self.listener.results.clear()
 
     def setUpCluster(self, scenario_dict):
-        assets = [
-            (scenario_dict["database_name"], scenario_dict["collection_name"]),
-            (scenario_dict["database2_name"], scenario_dict["collection2_name"]),
-        ]
+        assets = [(scenario_dict["database_name"],
+                   scenario_dict["collection_name"]),
+                  (scenario_dict.get("database2_name", "db2"),
+                   scenario_dict.get("collection2_name", "coll2"))]
         for db, coll in assets:
             self.client.drop_database(db)
             self.client[db].create_collection(coll)
 
+    def setFailPoint(self, scenario_dict):
+        fail_point = scenario_dict.get("failPoint")
+        if fail_point is None:
+            return
+
+        fail_cmd = SON([('configureFailPoint', 'failCommand')])
+        fail_cmd.update(fail_point)
+        client_context.client.admin.command(fail_cmd)
+        self.addCleanup(
+            client_context.client.admin.command,
+            'configureFailPoint', fail_cmd['configureFailPoint'], mode='off')
+
+    def assert_list_contents_are_subset(self, superlist, sublist):
```

Comment:
I added these methods to the class because now we can raise much better error messages when the equality/subset checks herein fail. Earlier, everything just raised `AssertionError` and print the same code where the error happened which wasn't helpful.

Reply:
Acknowledged.